api/tables/store.js

```diff
@@ -67,6 +70,61 @@ export function createStoreTable (region, tableName, options = {}) {
       await dynamoDb.send(new PutItemCommand(params))
   
       return item
+    },
+    /**
+     * Unbinds a link CID to an account
+     *
+     * @param {string} uploaderDID
+     * @param {string} payloadCID
+     */
+     remove: async (uploaderDID, payloadCID) => {
+      const params = {
+        TableName: tableName,
+        Key: marshall({
+          uploaderDID: uploaderDID.toString(),
+          payloadCID: payloadCID.toString(),
```

Comment:
Already strings according to the method params?

Reply:
good catch, I moved things around and forgot these